### PR TITLE
ioutil is deprecated

### PR DESCRIPTION
**Pull request by ramonskie**
as we moved to go 1.19 for the packr to embed pr #549 
ioutil is deprecated since go 1.17
and we need to use there io/os counter part packages

first pr #549 needs to be merged before this

### CODE REVIEW

File: acceptance-tests/bbl/upgrade_test.go [hunk 1]
```diff
@@ -94,7 +93,7 @@ var _ = Describe("Upgrade", func() {
 		Expect(err).NotTo(HaveOccurred())
 	})
 
-	It("is able to upgrade from an environment bbl'd up with an older version of bbl", func() {
+	FIt("is able to upgrade from an environment bbl'd up with an older version of bbl", func() {
```
Comment (selzoc) [inline acceptance-tests/bbl/upgrade_test.go:96]: Looks like there's still a test in focus here

**Review by selzoc — CHANGES_REQUESTED** [review]
There seems to be a bunch of changes unrelated to the deprecation of `ioutil`.  Looking over the `ioutil` replacement portions, looks good!  Just that one focused test to fix.

File: acceptance-tests/bbl/upgrade_test.go (hunk 1 above)
Comment (ramonskie) [reply acceptance-tests/bbl/upgrade_test.go:96]: the Fit is a mistake and will solve it.
the other changes are due to the fact that #549 is not merged yet

**Review by selzoc — APPROVED** [review]

**Comment by ramonskie** [comment]
rebased pr so it contains only the ioutil

**Review by ystros — APPROVED** [review]
Looks quite straightforward. Thanks!

**Review by rkoster — APPROVED** [review]

**Pull request merged**